--- a/src/setvtrgb.c	2018-10-30 12:01:10.095911415 +0530
+++ a/src/setvtrgb.c	2018-10-30 11:44:52.764897496 +0530
@@ -5,7 +5,6 @@
 #include <sys/ioctl.h>
 #include <linux/kd.h>
 #include <errno.h>
-#include <error.h>
 #include "kbd.h"
 #include "getfd.h"
 #include "nls.h"
@@ -60,8 +59,7 @@
 
 	/* Apply the color map to the tty via ioctl */
 	if (ioctl(fd, PIO_CMAP, colormap) == -1)
-		error(EXIT_FAILURE, errno, "ioctl");
-
+	perror("setvtrgb: ioctl"); exit(EXIT_FAILURE);
 	close(fd);
 }
 
@@ -72,7 +70,7 @@
 	unsigned int rows, cols, val;
 
 	if ((cmap = calloc(3 * 16, sizeof(unsigned char))) == NULL)
-		error(EXIT_FAILURE, errno, "calloc");
+	perror("setvtrgb: ioctl"); exit(EXIT_FAILURE);
 
 	for (rows = 0; rows < 3; rows++) {
 		cols = 0;
@@ -80,7 +78,7 @@
 		while (cols < 16) {
 			if ((c = fscanf(fd, "%u", &val)) != 1) {
 				if (c == EOF)
-					error(EXIT_FAILURE, errno, "fscanf");
+					perror("setvtrgb: ioctl"); exit(EXIT_FAILURE);
 
 				error(EXIT_FAILURE, 0, _("Error: %s: Invalid value in field %u in line %u."),
 				      filename, rows + 1, cols + 1);
@@ -141,7 +139,7 @@
 
 	} else {
 		if ((fd = fopen(file, "r")) == NULL)
-			error(EXIT_FAILURE, errno, "fopen");
+			perror("setvtrgb: ioctl"); exit(EXIT_FAILURE);
 
 		parse_file(fd, file);
 		fclose(fd);
